src/windows/minidump_writer.rs

```diff
@@ -13,8 +13,6 @@ pub struct MinidumpWriter {
     crash_context: crash_context::CrashContext,
     /// Handle to the crashing process, which could be ourselves
     crashing_process: HANDLE,
-    /// The pid of the crashing process.
-    crashing_pid: u32,
     /// The `EXCEPTION_POINTERS` contained in crash context is a pointer into the
```

Comment:
This was a slight refactoring from the bump to `crash-context 0.2` since the pid is now part of the crash-context itself on Windows to match linux and mac, so this allowed us to just cleanup the interface a bit since it can internally open the process in the correct way without the user needing to specify it.